_sass/default.scss

```diff
@@ -6,6 +6,11 @@ body {
     background: rgb(212, 212, 212);
 }
 
+h1 {
+    text-align: center;
+    padding-top: 20px;
```

Comment:
Use the existing spacing classes instead. See https://getbootstrap.com/docs/5.0/utilities/spacing/